lib/options.py

```diff
@@ -256,7 +256,7 @@ def __init__(self):
         parser.add_argument(
                 "--vardir",
                 dest="vardir",
-                default="var",
+                default='/tmp/t',
```

Comment:
Let's also change default in the help message.

Reply:
Fixed.